lib/katello/content/file.rb

```diff
@@ -0,0 +1,4 @@
+Katello::ContentPluginManager.register(::Katello::Repository::FILE_TYPE) do
+  description "File Repository types"
+  allow_creation_by_user false
```

Comment:
> I was more just curious the reasoning for locking users out of this type of repository. I didn't know if this was an implementation restriction or a conscious choice on our part to not let users do this for a particular reason (e.g. no use case, pointless, only applies in very limited scope, etc.)

This is just porting the current functionality. We don't allow one to create a 'file' type because so far it has not been useful